components/common/feeds/PostThread.tsx

```diff
@@ -66,7 +66,7 @@ export function ThreadPost({
         <ThreadPostWrapper depth={depth} highlight={highlight}>
           <PostAsCard
             className="bg-transparent px-3 py-2"
-            item={thread}
+            item={{ post: thread.post }}
```

Comment:
nit: It would probably be "cleaner" to change the props of `PostAsCard` to be:

```
  post: PostView
  reply?: ReplyRef
  reason?: $Typed<ReasonRepost> | $Typed<ReasonPin> | { $type: string }
  feedContext?: string
```

instead of `item: AppBskyFeedDefs.FeedViewPost` (I mean, this here feels kinda odd)